node_package/src/ReactOnRails.ts

```diff
@@ -204,7 +205,7 @@ ctx.ReactOnRails = {
    * Used by server rendering by Rails
    * @param options
    */
-  serverRenderReactComponent(options: RenderParams): string {
+  serverRenderReactComponent(options: RenderParams): null | string | Promise<RenderResult> {
```

Comment:
The types here don't actually match index.js. I'm not sure why TS compiler didn't complain. I'll fix in the morning.